components/x-teaser/src/concerns/image-service.js

```diff
@@ -1,4 +1,3 @@
-const { URL, URLSearchParams } = require('url')
```

Comment:
Are these imports no longer required? I know they exist in the browser but I'm not where this file is being executes, in Node or the browser.

Reply:
Nope, importing them was making errors get thrown in storybook components

<img width="1433" alt="Screenshot 2020-11-30 at 13 34 44" src="https://user-images.githubusercontent.com/20328348/100616305-d0c50c00-3310-11eb-8d0b-65184e6eb7ab.png">